CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -1037,7 +1042,13 @@ void IRenderWebBrowser.OnPopupShow(bool isOpen)
         /// <param name="isOpen">if set to <c>true</c> [is open].</param>
         protected virtual void OnPopupShow(bool isOpen)
         {
-            UiThreadRunAsync(() => { popupImage.Visibility = isOpen ? Visibility.Visible : Visibility.Hidden; });
+            UiThreadRunAsync(() => {
```

Comment:
Opening brace should be on new line for consistency.

Reply:
Done.